src/main/java/htsjdk/variant/vcf/VCFHeader.java

```diff
@@ -593,6 +627,30 @@ public VCFHeaderLine getOtherHeaderLine(final String key) {
         return mOtherMetaData.values();
     }
 
+    /**
+     * @param key    the header key or field type
+     * @param id     the header id
+     * @return the meta data line, or null if there is none
+     */
+    public VCFSimpleHeaderLine getIdHeaderLine(final String key, final String id) {
+        if (mIdMetaData.containsKey(key)) {
+            return mIdMetaData.get(key).get(id);
+        } else {
+            return null;
+        }
+    }
+
+    /**
+     * Returns the ID HeaderLines, in order by:
+     * - First key/field type to be added
+     * - Within key/field types, in original order
+     */
+    public Collection<VCFSimpleHeaderLine> getIdHeaderLines() {
+        final Collection<VCFSimpleHeaderLine> lines = new LinkedList<>();
+        mIdMetaData.forEach((k, v) -> lines.addAll(v.values()));
+        return lines;
+    }
+
```

Comment:
This class already has a public method `List<VCFIDHeaderLine> getIDHeaderLines()` that returns all ID lines. I think it will be confusing to also have this new method, with a slightly different name and signature that returns only the ID header lines that are not INFO, FORMAT, etc.